.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
You can add this to your global .gitignore instead of putting it in every repo you send a PR to.